pkg/serviceprovider/serviceprovider.go

```diff
@@ -43,7 +43,7 @@ type ServiceProvider interface {
 	// LookupToken tries to match an SPIAccessToken object with the requirements expressed in the provided binding.
 	// This usually searches kubernetes (using the provided client) and the service provider itself (using some specific
 	// mechanism (usually an http client)).
-	LookupToken(ctx context.Context, cl client.Client, binding *api.SPIAccessTokenBinding) (*api.SPIAccessToken, error)
+	LookupToken(ctx context.Context, cl client.Client, binding *api.SPIAccessTokenBinding) ([]api.SPIAccessToken, error)
```

Comment:
Could we change the name to `LookupTokens` and/or adjust the documentation above?